### DISCUSSION TITLE
Erro com a string Title

### DISCUSSION THREAD

**Discussion by Mathows** [balta-io/forum#318] [Tira Dúvidas] [answered]
Class Category

using System;

namespace BaltaDataAccess.Models
{
    public class Category
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
    }
}

Class Program

using System;
using BaltaDataAccess.Models;
using Dapper;
using Microsoft.Data.SqlClient;

namespace BaltaDataAccess
{
    class Program
    {
        private static void Main(string[] args)
        {
            const string connectionString = "Server=MAT-NOTE\\SQLEXPRESS;Database=balta;Integrated Security=SSPI";

         using(var connection = new SqlConnection(connectionString))
         {
            Console.WriteLine("Conectado");
            connection.Open();
            
            using(var command = new SqlCommand())
            {
               
                var categories = connection.Query<Category>("SELECT [Id], [Title] FROM [Categoty]");
                foreach(var category in categories)
                {
                    Console.WriteLine("${category.Id} - {category.Title}");
                }
            }

         }

            Console.WriteLine("Hello, World!");
        }
    }
}

Erro
C:\dev\BaltaDataAccess\Models\Category.cs(8,23): warning CS8618: O propriedade não anulável 'Title' precisa conter um valor não nulo ao sair do constr
utor. Considere declarar o propriedade como anulável.

**Comment by Mathows** [comment, marked as answer]
Encontrei o erro

**Reply by andrebaltieri** [reply]
Posta como resolveu por favor, assim pode ajudar mais pessoas!!
